src/index.js

```diff
@@ -1,32 +1,40 @@
+import {
+  add, editDescription, load, remove, save,
+} from './add-and-remove';
 import './style.css';
 
-const todo = [
-  {
-    description: 'List structure.',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Interactive list.',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Add and remove items.',
-    completed: false,
-    index: 2,
-  },
-];
+const todo = load() || [];
 
 function update() {
-  document.getElementById('list').innerHTML = todo.sort((a, b) => a.index - b.index).map((item, i) => `
+  const list = document.getElementById('list');
+  list.innerHTML = todo.map((item, i) => `
   <li class="todo-li">
     <input type="checkbox" id="chk-${i}" ${item.completed ? 'checked' : ''}>
     <label for="chk-${i}"></label>
     <input type="text" id="txt-${i}" value="${item.description}" >
     <label for="txt-${i}">${item.description}</label>
     <span class="handle"></span>
+    <button class="delete"></button>
   </li>`).join('');
+  list.querySelectorAll('li').forEach((li, i) => {
+    const element = todo[i];
+    li.querySelector('.delete').addEventListener('mousedown', () => {
+      remove(element, todo);
+      update();
+    });
+    li.querySelector(`#txt-${i}`).addEventListener('blur', (e) => {
+      editDescription(element, e.target.value);
+      save(todo);
+      update();
+    });
+  });
```

Comment:
- [ ] When I add a new task to the to-do list, the page is refreshing. Please also fix it by removing it 👌